lib/src/extensions.dart

```diff
@@ -269,3 +317,33 @@ extension NullableCallback3<A, B, C> on void Function(A, B, C)? {
   VoidCallback? safeVoidCall(A a, B b, C c) =>
       this == null ? null : () => this!(a, b, c);
 }
+
+extension BuildContextExtension on BuildContext {
+  MonthViewTheme get monthViewColors =>
```

Comment:
Add documentation comment. and add note that to make this work, add respective theme extension while initializing ThemeData.

Use comment template to avoid comment duplication.